core/src/test/scala/integration/kafka/network/DynamicConnectionQuotaTest.scala

```diff
@@ -317,6 +325,12 @@ class DynamicConnectionQuotaTest extends BaseRequestTest {
     }
   }
 
+  // make sure the connection count state is the same as the expectedConnectionCount
```

Comment:
nit: I would remove this comment. I think that the method is self-explanatory now.